python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -0,0 +1,267 @@
+"""CSC format sampling graph."""
+# pylint: disable= invalid-name
+from typing import Dict, Optional, Tuple
+
+import torch
+
+
+class GraphMetadata:
```

Comment:
Could consider refactor this to a pydantic object for saving/loading to/from JSON later. @Rhett-Ying